### PR TITLE
Export card configs can now have attached files.

### PR DESCRIPTION
**Pull request by manwithtwowatches**

### CODE REVIEW

File: app/controllers/export_card_configurations_controller.rb [hunk 1]
```diff
@@ -46,7 +46,11 @@ def new
 
   def create
     @config = ExportCardConfiguration.new(export_card_configurations_params)
+    attachments = @config.attach_files(params[:attachments])
+
     if @config.save
+      # Attachment.attach_files(@config, params[:attachments])
```
Comment (ulferts) [inline app/controllers/export_card_configurations_controller.rb]: this one is commented out, please remove

File: app/models/export_card_configuration.rb [hunk 2]
```diff
@@ -125,4 +126,15 @@ def can_activate?
   def can_deactivate?
     self.active && !is_default?
   end
+
+  # Required for attachments
+  def attachments_deletable?(usr=User.current)
+    # TODO RS: Authorisation
```
Comment (ulferts) [inline app/models/export_card_configuration.rb]: I believe this is limited to the admin-

File: app/models/export_card_configuration.rb [hunk 3]
```diff
@@ -125,4 +126,15 @@ def can_activate?
...
+    true
+  end
+
+  def attachments_visible?(user)
+    # TODO RS: Authorisation
```
Comment (ulferts) [inline app/models/export_card_configuration.rb]: There is probably no authorisation needed for that

**Comment by manwithtwowatches** [comment]
@ulferts Can you leave this for now? On closer inspection I don't think that the existing attachments framework is even good enough for what we want to do here. I will need some more time to work on this.

**Pull request closed without merging**